setup.py

```diff
@@ -24,7 +24,7 @@
         'fiona>=1.8.4,<2.*',
         'folium>=0.6.0',
         'ipyleaflet!=0.8.2',
-        'pyproj==1.9.6',
+        'pyproj',
```

Comment:
What about https://hynek.me/articles/conditional-python-dependencies/#pep-508?

```suggestion
        'pyproj<2; python_version<='3.4'',
        'pyproj; python_version>'3.5'',
```